uint32 iteration
float64 last_reward
float64 total_reward
